src/System.Memory/tests/Performance/Perf.ReadOnlySequence.Enumerator.cs

```diff
@@ -14,7 +14,7 @@ public class Perf_ReadOnlySequence_Enumerator
         private const int InnerCount = 10_000;
         volatile static int _volatileInt = 0;
 
-        [Benchmark(InnerIterationCount = InnerCount)]
+        [Benchmark(InnerIterationCount = InnerCount * 10)]
```

Comment:
I didn't realize almost all of the benchmark tests would require multiply by 10.

Change `private const int InnerCount = 10_000;` to 100_000 instead so you don't have to have multiply by 10 everywhere. And for the multisegment case, just divide by 10.

**Question:** What happens if we keep the multisegment InnerCount as 100_000 as well? Does the test end up taking too long (over 1 second)?

Reply:
It takes ten times longer that others tests. I want to see similar values to compare method speed.